client/sites/components/panel-sidebar/style.scss

```diff
@@ -18,20 +18,33 @@
 	list-style: none;
 }
 
-.panel-sidebar-tab {
+.components-button.panel-sidebar-tab {
 	border-radius: 4px;
 	width: 100%;
-	margin: 2px 0;
```

Comment:
Maybe:
```
margin: 0 0 2px 0;
```
Without it, mousing over the item below looks kind of blocky. I think we want a little bit of space:

![Screenshot_2024-11-01_11-52-08](https://github.com/user-attachments/assets/a9d02746-9ec4-4177-ad1c-bc92be891012)
![Screenshot_2024-11-01_11-51-47](https://github.com/user-attachments/assets/621eb7b3-ba99-45d7-8de2-e8fce57298f1)

Reply:
Oops, I missed that the Figma do actually contain the margin 🤦‍♂️ added it!